### PR TITLE
Added Slovak language support #12

### PR DESCRIPTION
**Pull request by Blarne**

### CODE REVIEW

File: src/main/java/pl/allegro/finance/tradukisto/internal/languages/MissingFormException.java [hunk 1]
```diff
@@ -1,6 +1,9 @@
 package pl.allegro.finance.tradukisto.internal.languages;
 
 public class MissingFormException extends RuntimeException {
+
+    private static final long serialVersionUID = 1L;
```
Comment (dorians) [inline src/main/java/pl/allegro/finance/tradukisto/internal/languages/MissingFormException.java]: This is unnecessary.

File: src/main/java/pl/allegro/finance/tradukisto/internal/languages/slovak/SlovakValues.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        return "€";
+    }
+
+    @Override
+    public char twoDigitsNumberSeparator() {
+        return ' ';
+    }
+
+    public Map<Integer, String> exceptions() {
+        return ImmutableMap.<Integer, String>builder().put(2, "dve").build();
```
Comment (dorians) [inline src/main/java/pl/allegro/finance/tradukisto/internal/languages/slovak/SlovakValues.java]: We moved to Java 8 so you can remove generic types here.

Comment (Blarne) [reply src/main/java/pl/allegro/finance/tradukisto/internal/languages/slovak/SlovakValues.java]: I don't understand, see other classes (CzechValues, GermanValues), implementation is same, can't remove generics for ImmutableMap. Can you provide some code or change it on pull-requtest merge?

`GermanValues.java`

```java
public Map<Integer, String> exceptions() {
        return ImmutableMap.<Integer, String>builder().put(1, "eins").build();
}
```

File: src/main/java/pl/allegro/finance/tradukisto/internal/languages/MissingFormException.java (hunk 1 above)
Comment (Blarne) [reply src/main/java/pl/allegro/finance/tradukisto/internal/languages/MissingFormException.java]: OK, removed.

**Comment by Blarne** [comment]
Some progress in merge request accept?

**Comment by dorians** [comment]
I created pull request with tests for Slovak languages. Could you take a look at it and review it?

**Comment by dorians** [comment]
I've released new version of Tradukisto which includes your changes!
https://mvnrepository.com/artifact/pl.allegro.finance/tradukisto/1.2.0

Enjoy! :)

**Pull request merged**